packages/suite/package.json

```diff
@@ -23,13 +23,16 @@
         "@trezor/suite-data": "^1.0.0",
         "@trezor/suite-storage": "^1.0.0",
         "@trezor/typescript-typings": "^1.0.0",
+        "@types/debounce": "^1.2.0",
+        "@types/jest": "^24.0.13",
```

Comment:
those should be placed in "top-level" package.json (see yarn lock, now there are 2 @types/jest packages)